projects/rawkode.academy/platform/web/.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
Why do we need this?

Reply:
Yes! When you have a system-wide config which overrides this, the content of `bun.lockb` changes

Reply:
Did you have a system wide config? 😛